src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
+@use "./blocks/card.scss";
+@use "./blocks/stars.scss";
+@use "./utils/variables";
+
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$grey-elements: #f3f3f3;
+$blue-accent: #00acdc;
+$font-regular: 400;
+$font-medium: 500;
+$font-bold: 700;
```

Comment:
Why you are declaring variables here and in /utils/variables? Do it only one time to avoid confusion.

Alos there is no need for $font-medium/bold/regular variable. It is more readable to put it directly in `font-weight` property